src/main/java/RuntimeSupport/DukeException.java

```diff
@@ -0,0 +1,36 @@
+package RuntimeSupport;
+
+public class DukeException {
+
+    static String BREAK_LINE = "____________________________________________________________";
+    public static void handleException(Exception error, String line) {
+
+        if (error instanceof NullPointerException) {
+
+            System.out.println("Alert! Galactic navigation systems report a phantom destination \uD83C\uDF0C. The operation: " + line + " cannot be done because the event does not exist!");
```

Comment:
You could consider splitting lines with over 120 characters to 2 separate lines to adhere to the java coding [standard](https://se-education.org/guides/conventions/java/basic.html)